src/documentation/ntfs2gtfs.md

```diff
@@ -0,0 +1,147 @@
+
+#### routes.txt
+
+GTFS field | Required | NTFS file | NTFS field | Note
+--- | --- | --- | --- | ---
+route_id | yes | lines.txt | line_id | 
+agency_id | no | lines.txt | network_id | (link to the [agency.txt](#agency.txt) file)
+route_short_name | yes | | lines.txt | line_code | 
+route_long_name | yes | lines.txt | line_name | 
+route_type | yes | | | The corresponding physical mode of the trips of the line, otherwise the default value is `3`. See the table below for the mapping of modes.
+route_color | no | lines.txt | line_color | 
+route_text_color | no | lines.txt | line_text_color | 
+route_sort_order | no | lines.txt | line_sort_order | 
+
+Each line of this file corresponds to a transit line modeled in the GTFS feed. In case a transit line uses more than one modes of transportation, it should be modeled separately for each different mode, according to the mapping of modes presented below. The priorities follow the [NeTex Specification](http://www.normes-donnees-tc.org/wp-content/uploads/2014/05/NF_Profil_NeTEx_pour_les_arrets-_F-_-_v2.pdf) (cf. chapter 6.2.3). 
+- The trips using the physical mode with the lowest priority are modeled by a GTFS line with the field `route_id` matching the value of `line_id`.
+- The trips using other physical modes are modeled by separate GTFS lines, adding the suffix ":<physical_mode_id>" to the value of `route_id` and assigning the corresponding physical mode to the field `route_type`.
+
+physical_mode_id in the NTFS | route_type in the GTFS | Priority w.r.t. NeTex
+--- | --- | ---
+RailShuttle | 0 | 3
+Tramway | 0 | 5
+Metro | 1 | 4
+LocalTrain | 2 | 3
+LongDistanceTrain | 2 | 3
+RapidTransit | 2 | 3
+Train | 2 | 3
+Other (or unknown) | 3 | 7
+Bus | 3 | 7
+BusRapidTransit | 3 | 7
+Coach | 3 | 7
+Boat | 4 | 2
+Ferry | 4 | 2
+Funicular | 7 | 6
+Shuttle | 7 | 6
+
+### stops.txt
+The entrances/exits of stations (stops having `location_type` = 2) are not handled in the current version.
+
+GTFS field | Required | NTFS file | NTFS field
+--- | --- | --- | --- 
+stop_id | yes | stops.txt | stop_id 
+stop_name | yes | stops.txt | stop_name 
+stop_lat | yes | stops.txt | stop_lat 
+stop_lon | yes | stops.txt | stop_lon 
+zone_id | no | stops.txt | fare_zone_id
+location_type | no | stops.txt | location_type
+parent_station | no | stops.txt | parent_station 
+
+### trips.txt
+
+GTFS field | Required | NTFS file | NTFS field | Note
+--- | --- | --- | --- | ---
+route_id | yes | trips.txt | route_id | 
+service_id | yes | trips.txt | service_id | 
+trip_id | yes | trips.txt | trip_id | 
+trip_headsign | no | trips.txt | | (1)
+trip_short_name | no | trips.txt | | (1)
+trip_desc | no | comments.txt, comment_links.txt | | The value of `comment_name` referenced by the `comment_id` having an `object_type` = `trip` and an `object_id` equal to the corresponding `trip_id`. In case of more than one comments linked to the same trip, the first one in alphabetical order is taken into account.
```

Comment:
It' a hard one. I think you should add `comment_name` in the `NTFS field` column.
I wonder if the part `the first one in alphabetical order` is ambiguous. is it the `comment_name` in alphabétical order ? Can you make it explicit ?